chef/site-cookbooks/wca/recipes/regulations.rb

```diff
@@ -1,23 +1,26 @@
 username, repo_root = WcaHelper.get_username_and_repo_root(self)
 
 #### Regulations dependencies
-# Dependencies for regulations. Some of this came from
-# https://github.com/cubing/wca-documents-extra/blob/master/.travis.yml, but has
-# been tweaked for Ubuntu 14.04
 package "git"
-package "texlive-fonts-recommended"
-package "pandoc"
+package "python-pip"
 package "fonts-unfonts-core"
-package "fonts-arphic-uming"
-package "texlive-lang-all" do
-  # This package takes *forever* to install. Give it an hour.
-  timeout 60*60
-end
-package "texlive-xetex"
-package "texlive-latex-recommended"
-package "texlive-latex-extra"
+package "fonts-wqy-microhei"
 package "lmodern"
 
+bash "install_wkhtmltopdf" do
+  not_if "/usr/local/bin/wkhtmltopdf --version | grep -q 'with patched qt'"
+  user "root"
+  cwd "/tmp"
+  code <<-EOH
+    wget http://download.gna.org/wkhtmltopdf/0.12/0.12.3/wkhtmltox-0.12.3_linux-generic-amd64.tar.xz -O wkhtml.tar.xz
+    tar -xf wkhtml.tar.xz -C /usr/local
+    EOH
+end
+
+execute "pip install wrc --upgrade" do
+  user "root"
+end
+
 execute "#{repo_root}/scripts/deploy.sh rebuild_regs" do
   user username
   not_if "ls #{repo_root}/WcaOnRails/app/views/regulations"
```

Comment:
Is this executed every time the `production` branch is updated, or only once when the server environment is built?